src/code_8057824_1.c

```diff
@@ -3815,7 +3815,8 @@ void CreateJirachiWishWarpTile(void)
 {
   struct Tile *tile;
 
-  tile = GetTileSafe(gDungeon->unkE23C, gDungeon->unkE23E);
+  tile = GetTileSafe(gDungeon->unkE220[7].x, gDungeon->unkE220[7].y);
+  //tile = GetTileSafe(gDungeon->unkE23C, gDungeon->unkE23E);
```

Comment:
Are these commented-out lines still needed?

Reply:
Nope, will remove. Had them pre-commit in case it didn't match.

Reply:
Removed